packages/integration-testsuite/src/apolloServerTests.ts

```diff
@@ -843,7 +843,7 @@ export function defineIntegrationTestSuiteApolloServerTests(
             },
           },
           introspection: true,
-          includeStackTracesInErrorResponses: true,
+          includeStacktraceInErrorResponses: true,
```

Comment:
I think I'm going to delete this test. Reviewing the initial PR that introduced it (#1288) I don't understand what was interesting about this random `yup` package and suspect it is related to this flattening code which we have deleted. The PR description refers to schema-stitching stuff too. Since the error code is so much simpler now I doubt we have to test one random style of error.